Source.c

```diff
@@ -0,0 +1,101 @@
+#include<stdio.h>
+#include<math.h>
+#include<conio.h>
```

Comment:
it is not good to use dos lib "conio.h " by microsoft only if you need getch()  for pause in Unix prog.And also you dont use it at all delete this